spot-electron/src/logger/utils.js

```diff
@@ -0,0 +1,30 @@
+const os = require('os');
+
+/**
+ * Log levels map.
+ */
+const LOG_LEVEL = {
+    0: 'TRACE',
+    1: 'DEBUG',
+    2: 'INFO',
+    3: 'LOG',
+    4: 'WARN',
+    5: 'ERROR'
+};
+
+module.exports = {
+    /**
+     * Builds the log message.
+    */
+    parseLogMessage: (level, message) => `[${LOG_LEVEL[level]}] ${message}`,
+
+    /**
+     * Returns the host identifier.
+     */
+    getHostId: () => {
+        const ipAddress = [].concat(...Object.values(os.networkInterfaces()))
```

Comment:
Why do we want to have the IP address here?

Reply:
In the case of cloudwatch, I use it as the log stream name, and for s3 I prefix the logs with it. It's all in order to identify which machine was running the meeting. Maybe there's a better way to achieve this, please advise if so.